backend/src/Tabulators/Star.ts

```diff
@@ -171,62 +171,83 @@ function runStarRound(summaryData: summaryData, remainingCandidates: candidate[]
-      roundResults.logs.push(`${scoreTieWinners[0].name} wins score tiebreaker and advances to the runoff round.`)
-      continue
-    }
-    if (scoreTieWinners.length === 2 && runoffCandidates.length === 0) {
+
+    if (scoreWinners.length === 2 && nCandidatesNeeded === 2) {
       // Tie between two candidates, but both can advance to runoff
-      runoffCandidates.push(...scoreTieWinners)
-      roundResults.logs.push(`${scoreTieWinners[0].name} and ${scoreTieWinners[1].name} win score tiebreaker and advances to the runoff round.`)
-      continue
+      runoffCandidates.push(...scoreWinners)
+      roundResults.logs.push(`${scoreWinners[0].name} and ${scoreWinners[1].name} advance to the runoff round.`)
+      continue scoreLoop
     }
-    // True Tie, other tiebreaker is neededed
-    roundResults.logs.push(`True tie between ${scoreTieWinners.map(c => c.name).join(', ')}.`)
-    if (enablefiveStarTiebreaker) {
-      // Five-star tiebreaker is enabled, look for candidate with most 5 star votes
-      const fiveStarWinners = fiveStarTiebreaker(summaryData, scoreTieWinners)
-      if (fiveStarWinners.length === 1) {
-        // Candidate wins five-star tiebreaker
-        runoffCandidates.push(...fiveStarWinners)
-        roundResults.logs.push(`${scoreTieWinners[0].name} wins five-star tiebreaker and advances to the runoff round.`)
-        continue
+    // Multiple candidates have top score, proceed to score tiebreaker
+    roundResults.logs.push(`${scoreWinners.map(c => c.name).join(', ')} advance to score tiebreaker.`)
+    let tiedCandidates = scoreWinners
+    tieLoop: while (tiedCandidates.length > 1) {
+      // Get candidates with the most head to head losses
+      let headToHeadLosers = getHeadToHeadLosers(summaryData, tiedCandidates)
+      if (headToHeadLosers.length < tiedCandidates.length) {
+        // Some candidates have more head to head losses than others, remove them from the tied candidate pool
+        roundResults.logs.push(`${headToHeadLosers.map(c => c.name).join(', ')} removed from score tiebreaker with the most losses.`)
+        tiedCandidates = tiedCandidates.filter(c => !headToHeadLosers.includes(c))
+        continue tieLoop
       }
-      if (fiveStarWinners.length === 2 && runoffCandidates.length === 0) {
-        // Two candidates win five-star tiebreaker, both advance to runoff
-        runoffCandidates.push(...fiveStarWinners)
-        roundResults.logs.push(`${fiveStarWinners[0].name} and ${fiveStarWinners[1].name} win five-star tiebreaker and advances to the runoff round.`)
-        continue
+      // All tied candidates have the same number of head to head lossess
+      if (nCandidatesNeeded === 2 && tiedCandidates.length === 2) {
+        // Tie between two candidates, but both can advance to runoff
+        runoffCandidates.push(...tiedCandidates)
+        roundResults.logs.push(`${tiedCandidates[0].name} and ${tiedCandidates[1].name} advance to the runoff round.`)
```

Comment:
nit, maybe this log can be worded differently to distinguish it from line 188. That way we can tell if the tie was passed through during the tiebreaker phase instead of prior. Maybe "advanced to runoff round from tiebreaker"?

Reply:
Done